Adding HCP and HCP Terraform Links to make the platform easier to find (#856)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/browse.mdx

## Find a workspace

> 1. Sign in to [HCP Terraform](https://app.terraform.io/) or Terraform Enterprise and choose the organization you want to browse.

Reviewer: ```suggestion
1. Sign in to [HCP Terraform](https://app.terraform.io/) or Terraform Enterprise and select your organization.
```
